docs/core/deploying/acquisition-experiences.md

```diff
@@ -0,0 +1,171 @@
+
+|Acquisition Choice|Acquire|Update|
+|:-|:-:|:-:|
+[Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)   | ✔️ | ✔️ |
+[https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)        | ✔️ | ❌ |
+[Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)                | ✔️ | ✔️ |
+[Docker](#Docker)                                                               | ✔️ | ✔️ |
+[Linux Package Managers](#Linux-Package-Managers)                               | ✔️ | ✔️\*|
+[Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)   | ✔️ | ✔️ |
+[Snap store](#Snap-store)                                                       | ✔️ | ✔️ |
+[.NET Core on GitHub](#NET-Core-on-GitHub)                                      | ✔️ | ❌ |
+
+_\*Some package Managers provide update notifications__
+
+Additionally, differences between these choices make some of these choices a great option for developers while others are better suited for DevOps and IT administrators.
+
+|Acquisition Choice|Great For Developers|Great For DevOps/ IT Admins|
+|:-|:-:|:-:|
+[Visual Studio and Visual Studio Code](#Visual-Studio-and-Visual-Studio-Code)   | ✔️ | ❌ |
+[https://dot.net (Microsoft .NET site)](#httpsdotnet-Microsoft-NET-site)        | ✔️ | ✔️ |
+[Dotnet-install.ps1/.sh Scripts](#Dotnet-install.ps1.sh-Scripts)                | ✔️ | ✔️ |
+[Docker](#Docker)                                                               | ✔️ | ✔️ |
+[Linux Package Managers](#Linux-Package-Managers)                               | ✔️ | ✔️ |
+[Azure DevOps pipeline Installer Task](#Azure-DevOps-pipeline-Installer-Task)   | ❌ | ✔️ |
+[Snap store](#Snap-store)                                                       | ✔️ | ✔️ |
+[.NET Core on GitHub](#NET-Core-on-GitHub)                                      | ✔️ | ❌ |
+
+
+Let's look at each of these choices for .NET Core acquisition in further detail.
+
+## Visual Studio and Visual Studio Code
+
+Developers often acquire and update the .NET Core tools and runtime by installing Visual Studio or Visual Studio for Mac. 
+
+### Visual Studio for Windows
+Each version of Visual Studio installs the latest .NET Core SDK tools, and one or more runtimes so you can target your app to run on the .NET Core runtime version of your choice.
+
+Selecting any of the managed workloads for Web, Desktop or Azure development will install the .NET Core development tools and runtime.
+
+![Install Visual Studio for Windows](media/install-vs-windows.png "Install Visual Studio for Windows")
+
+In recent previews of Visual Studio 2019 Update 3, we have made changes to the installation experience for .NET Core the experience more streamlined and efficient. More details about this can be found in the blog post [Improving .NET Core installation in Visual Studio and on Windows](https://devblogs.microsoft.com/dotnet/improving-net-core-installation-in-visual-studio-and-on-windows/). 
```

Comment:
```suggestion
In recent previews of Visual Studio 2019 Update 3, we have made changes to the installation experience for .NET Core in order to make the experience more streamlined and efficient. More details about this can be found in the blog post [Improving .NET Core installation in Visual Studio and on Windows](https://devblogs.microsoft.com/dotnet/improving-net-core-installation-in-visual-studio-and-on-windows/). 
```